packages/firestore/test/unit/util/async_queue.test.ts

```diff
@@ -137,7 +138,9 @@ describe('AsyncQueue', () => {
     });
   });
 
-  it('can schedule ops in the future', async () => {
+  // Flaky on Safari.
```

Comment:
Do we know why it is flaky on Safari?

Reply:
I haven't been able to find any reason. The test relies on two setTimeouts at 1ms and 5ms happening in the correct order, and sometimes Safari doesn't seem to get the order right. I can't find any known bug or issue that might cause that.